### PR TITLE
Integration test for Find Wallet By Id

### PR DESCRIPTION
**Pull request by zetxdev**

### CODE REVIEW

**Review by dawciobiel — CHANGES_REQUESTED** [review]
I removed .gitignore, sorry, you will have to commit this file again.
But please restore - just for the commit/push time - file content from the `develop` branch.

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindWalletByIdTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        walletRepository.deleteAll();
+    }
+
+    @DisplayName("It should return wallet DTO")
+    @Test
+    void testFindWalletByIdAPI_whenWalletIdIsCorrect_thenReturnWalletDTO() throws Exception {
+        Wallet wallet = walletRepository.save(new Wallet("Test Wallet"));
+        mockMvc.perform(get("/api/wallets/{id}", wallet.getId())
+                        .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
```
Comment (DawidStuzynski) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindWalletByIdTest.java:34]: We need to consider it, but in my opinion this line would be great for debugging only

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindWalletByIdTest.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                .andExpect(MockMvcResultMatchers.jsonPath("$.name").value(wallet.getName()));
+        Assertions.assertEquals(1, walletRepository.count());
+
+    }
+
+    @DisplayName("It should return status Not Found and AppRuntimeException")
+    @Test
+    void testFindWalletByIdAPI_whenWalletIdIsIncorrect_thenReturnErrorResponse() throws Exception {
+        mockMvc.perform(MockMvcRequestBuilders.get("/api/wallets/{id}", 1L).accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
```
Comment (DawidStuzynski) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindWalletByIdTest.java:46]: We need to consider it, but in my opinion this line would be great for debugging only

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindWalletByIdTest.java (hunk 1 above)
Comment (zetxdev) [reply src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindWalletByIdTest.java:34]: Indeed, that was for debugging, to see visually what should be returned. It is not critical for the test, so if you want me to remove it I can do it

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindWalletByIdTest.java (hunk 2 above)
Comment (zetxdev) [reply src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindWalletByIdTest.java:46]: Indeed, that was for debugging, to see visually what should be returned. It is not critical for the test, so if you want me to remove it I can do it

**Review by dawciobiel — DISMISSED** [review]
looks good to me

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindWalletByIdTest.java (hunk 1 above)
Comment (odzio33) [reply src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindWalletByIdTest.java:34]: I agree with DawidS, please remove print method. This is a good approach to debugging

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindWalletByIdTest.java (hunk 2 above)
Comment (odzio33) [reply src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindWalletByIdTest.java:46]: I agree with DawidS, please remove print method. This is a good approach to debugging

**Review by odzio33 — CHANGES_REQUESTED** [review]
Changes required
I agree with DawidS, please remove print method. This is a good approach to debugging

**Comment by zetxdev** [comment]
Removed andDo(print())

**Pull request closed without merging**